client/app/components/request-list/request-list.directive.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .directive('requestList', RequestListDirective);
+
+  /** @ngInject */
+  function RequestListDirective() {
+    var directive = {
+      restrict: 'E',
+      scope: {
+        'items': '<',
+        'config': '<?',
```

Comment:
It's the one way binding thing, like `=` and `=?` but doesn't let you change it from the directive.. essentially it takes that `items="vm.something"` and binds that to that directive's controller's `.items`